templates/sonar.properties

```diff
@@ -245,22 +299,99 @@ sonar.search.port={{ sonar_search_port }}
+#   Compute Engine (aka. CE) logs in ce.log
+#   Elasticsearch (aka. ES) logs in es.log
+#
+# All 4 files follow the same rolling policy (see sonar.log.rollingPolicy and sonar.log.maxFiles) but it applies
+# individually (eg. if sonar.log.maxFiles=4, there can be at most 4 of each files, ie. 16 files in total).
+#
+# All 4 files have logs in the same format:
+#           1           2    3           4                       5                                                   6
+# |-----------------| |---| |-|--------------------||------------------------------| |------------------------------------------------------------------------------------------------------------------------------|
+# 2016.11.16 16:47:00 INFO  ce[AVht0dNXFcyiYejytc3m][o.s.s.c.t.CeWorkerCallableImpl] Executed task | project=org.sonarqube:example-java-maven | type=REPORT | id=AVht0dNXFcyiYejytc3m | submitter=admin | time=1699ms
+#
+# 1: timestamp. Format is YYYY.MM.DD HH:MM:SS
+#    YYYY: year on 4 digits
+#    MM: month on 2 digits
+#    DD: day on 2 digits
+#    HH: hour of day on 2 digits in 24 hours format
+#    MM: minutes on 2 digits
+#    SS: seconds on 2 digits
+# 2: log level.
+#    Possible values (in order of descending criticality): ERROR, WARN, INFO, DEBUG and TRACE
+# 3: process identifier. Possible values: app (main), web (Web Server), ce (Compute Engine) and es (Elasticsearch)
+# 4: SQ thread identifier. Can be empty.
+#    In the Web Server, if present, it will be the HTTP request ID.
+#    In the Compute Engine, if present, it will be the task ID.
+# 5: logger name. Usually a class canonical name.
+#    Package names are truncated to keep the whole field to 20 characters max
+# 6: log payload. Content of this field does not follow any specific format, can vary in length and include line returns.
+#    Some logs, however, will follow the convention to provide data in payload in the format " | key=value"
+#    Especially, log of profiled pieces of code will end with " | time=XXXXms".
+
+# Global level of logs (applies to all 4 processes).
+# Supported values are INFO (default), DEBUG and TRACE
+#sonar.log.level=INFO
+
+# Level of logs of each process can be controlled individually with their respective properties.
+# When specified, they overwrite the level defined at global level.
+# Supported values are INFO, DEBUG and TRACE
+#sonar.log.level.app=INFO
+#sonar.log.level.web=INFO
+#sonar.log.level.ce=INFO
+#sonar.log.level.es=INFO
 
 # Path to log files. Can be absolute or relative to installation directory.
 # Default is <installation home>/logs
 #sonar.path.logs=logs
 
+# Rolling policy of log files
+#    - based on time if value starts with "time:", for example by day ("time:yyyy-MM-dd")
+#      or by month ("time:yyyy-MM")
+#    - based on size if value starts with "size:", for example "size:10MB"
+#    - disabled if value is "none".  That needs logs to be managed by an external system like logrotate.
+sonar.log.rollingPolicy=time:yyyy-MM-dd
```

Comment:
These could be configurable as well. You wouldn't have to define defaults in the defaults file if you don't want, you could just to `{{ sonar_log_rollingPolicy | default("time:yyyy-MM-dd") }}`

Reply:
yes, fixed